dbt/adapters/databricks/impl.py

```diff
@@ -109,7 +109,7 @@
 @dataclass
 class DatabricksConfig(AdapterConfig):
     file_format: str = "delta"
-    table_format: TableFormat = TableFormat.DEFAULT
+    table_format: str = TableFormat.DEFAULT
```

Comment:
on 3.12, mypy starts complaining that these enums actually inherit str, so their values are str.